task-management/src/main/webapp/skin/css/tasks.less

```diff
@@ -3018,8 +3019,10 @@
     }
     .changesText {
       max-width: 270px;
+      margin: 10px 6px;
+      padding-bottom: 0%;
```

Comment:
Can we use vue css helper in the vue file ?

Reply:
Yes we can use vuetify spacing